### PR TITLE
✨ drops page shortcuts

### LINKED ISSUE #9608: keyboard shortcut to create new variation
i think it would be nice to have easier keyboard access - going through variations with a single keystroke would be nice

- what keystroke would it be? enter? space? 

would need a design - this issue is not ready for grab

### PR DESCRIPTION
**Pull request by roiLeo**
Added 3 new keyboards shorcuts on drop page: 
- `v+n` => create new variation
- `v++` => add +1 item to mint
- `v+-`=> remove -1 item to mint

Let me know if you need any other shorcut that can be implemented.

## PR Type

- [x] Feature

## Needs Design check

- @exezbcz please review

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #9608 
- [x] Fix Modal Shortcut not showing

### CODE REVIEW

**Review by Jarsen136 — COMMENTED** [review]
It may be out of the scope of this issue, but the `[keyboardShortcutsModal.vue` modal does not work at all and would not be shown.

File: components/shared/modals/keyboardShortcutsModal.vue
[the remark this answers is not in the document]
Comment (roiLeo) [reply components/shared/modals/keyboardShortcutsModal.vue:1]: Screenshot 2024-07-17 at 11-59-53 Settings One Stop Shop for Polkadot NFTs

[the remark this answers is not in the document]
Comment (Jarsen136) [reply components/shared/modals/keyboardShortcutsModal.vue:1]: image

Could you also add a `drops` title here?

**Review by Jarsen136 — APPROVED** [review]
wfm

**Review by preschian — APPROVED** [review]

**Review by hassnian — APPROVED** [review]

**Comment by vikiival** [comment]
the shortcuts does not work for me 👀 

i tried to press `v` and `n` or `v` and `+` at it had no effect 
Mac, Chrome

**Comment by roiLeo** [comment]
> the shortcuts does not work for me 👀
> 
> i tried to press `v` and `n` or `v` and `+` at it had no effect Mac, Chrome

Tested on Mac, Chrome -> works for me

Do you hold 'v' & '+' key and not 'v' & '=' on mac keyboard?

**Comment by vikiival** [comment]
> Do you hold 'v' & '+' key and not 'v' & '=' on mac keyboard?

That was one issue,

but also `vn` hasn't worked before

Seems it works now

**Comment by hassnian** [comment]
@roiLeo let's get this merged, could you resolve the conflicts, tyvm

**Comment by hassnian** [comment]
@roiLeo can you fix this 
CleanShot 2024-08-15 at 17 15 10@2x

**Comment by roiLeo** [comment]
> @roiLeo can you fix this 
> CleanShot 2024-08-15 at 17 15 10@2x
> 

Already fixed?

**Comment by hassnian** [comment]
> Already fixed?

weird

thanks! 

@vikiival ready to merge

**Pull request merged**